PriorityQueue/include/PriorityQueue.h

```diff
@@ -1,19 +1,43 @@
 #pragma once
-
-#include <iostream>
 #include <list>
-#include <exception>
+#include <stdexcept>
+
+using empty_queue_except = std::out_of_range;
 
+template <typename T>
+using priorityQueue = std::list<T>;
```

Comment:
It's a bit confusing to the reader that you have two types called `p/PriorityQueue`. Maybe rename this one